plugin/pkg/scheduler/factory/factory.go

```diff
@@ -237,6 +240,29 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}, nil
 }
 
+func (f *ConfigFactory) getFitSchedulerPod() *api.Pod {
+	pod := f.PodQueue.Pop().(*api.Pod)
+	if f.podFitScheduler(pod) {
```

Comment:
I would suggest to rename the function "podFitScheduler" to something like "responsibleForPod"